apps/frontend/pages/lems/pit-admin.tsx

```diff
@@ -91,7 +93,8 @@ const Page: NextPage<Props> = ({
         action={
           <Stack direction="row" spacing={2}>
             <ConnectionIndicator status={connectionStatus} />
-            <ReportLink division={division} />
+            <DivisionDropdown event={division.event} selected={division._id.toString()} />
```

Comment:
Shouldn't this be conditional based on if this is indeed an event user for this event?